controllers.js

```diff
@@ -0,0 +1,12 @@
+const {getTopicModel} = require('./models')
+
+exports.getTopic = (req, res, next) => {
+    getTopicModel()
+    .then((topics) => {
+        res.status(200).send(topics);
+    })
+    .catch( (err) => {
+        next(err);

```

Comment:
nicely done to include the catch block!
If you like a cleaner way to write this in express is:  `.catch(next);`  - express will read this the same